models/yang/sonic/sonic-system-tacacs.yang

```diff
@@ -0,0 +1,100 @@
+module sonic-system-tacacs {
+    namespace "http://github.com/Azure/sonic-system-tacacs";
+    prefix ssys;
+
+    import ietf-inet-types {
+        prefix inet;
+    }
+
+    revision 2019-11-13 {
+        description "Initial revision.";
+    }
+
+    container sonic-system-tacacs {
+
+        container TACPLUS_SERVER {
+            list TACPLUS_SERVER_LIST {
+                key "ipaddress";
+
+                leaf ipaddress {
+                    type inet:ip-address;
+                }
+
+                leaf priority {
+                    type uint8;
+                }
+
+                leaf tcp_port {
+                    type inet:port-number;
+                }
+
+                leaf timeout {
+                    default 5;
+                    type uint16;
+                }
+
+                leaf auth_type {
+                    type enumeration {
```

Comment:
Define it separately and use it, instead of defining multiple times

Reply:
Done